### PR TITLE
Man 8943 serverless policy in terraform provider

### PR DESCRIPTION
**Pull request by rafaelseidelcisco**

### CODE REVIEW

File: internal/escher_api/model/cd_serverless_rule.go [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+
+	// id
+	// Format: uuid
+	ID strfmt.UUID `json:"id,omitempty"`
+
+	// name
+	// Required: true
+	Name *string `json:"name"`
+
+	ruleField ServerlessRuleType
```
Comment (bensagi) [inline internal/escher_api/model/cd_serverless_rule.go:42]: why serverlessRuleType is not a pointer as in serverlessRuleStatus?

File: internal/escher_api/model/cd_serverless_rule.go [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+
+	// scope
+	Scope []*ServerlessRuleScope `json:"scope"`
```
Comment (bensagi) [inline internal/escher_api/model/cd_serverless_rule.go:45]: why []* ?

**Review by bensagi — APPROVED** [review]

File: internal/escher_api/model/cd_serverless_rule.go (hunk 2 above)
Comment (rafaelseidelcisco) [reply internal/escher_api/model/cd_serverless_rule.go:45]: each scope is a cloud account + regions

File: internal/escher_api/model/cd_serverless_rule.go (hunk 1 above)
Comment (rafaelseidelcisco) [reply internal/escher_api/model/cd_serverless_rule.go:42]: this is how the auto generator generated it

**Pull request merged**